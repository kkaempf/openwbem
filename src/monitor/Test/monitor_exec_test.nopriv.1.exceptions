InsufficientPrivilegesException   CheckException: monitored.*Spawn: insufficient privileges
