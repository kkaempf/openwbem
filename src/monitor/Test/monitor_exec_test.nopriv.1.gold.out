Caught OpenWBEM::Exception:
  type: PrivilegeManagerException
  msg:  CheckException: monitoredSpawn: insufficient privileges
